### ISSUE TITLE
Error at the moment to initialize two notebooks

### ISSUE THREAD

**Issue by stroncod** [mljar/mercury#352]
I'm getting this error when I try to initialize the second notebook: 
```
TypeError: clean() got an unexpected keyword argument 'css_sanitizer'
```

I'm guessing there is some problems with Django. My notebooks does not use any widgets is an static notebook. This is the `App`
```python
# set Application parameters
app = mr.App(title="Pickle Output tutorial",
        description="Learn how to handle the outputs from the Scheduler Test",
        show_code=True,
        show_prompt=True,
        continuous_update=False,
        static_notebook=True,
        show_sidebar=False,
        full_screen=True,
        allow_download=True)

```

Notebook is running  fine in Pycharm.
Here is the traceback:

```
The notebook try_scheduler.ipynb will be updated
Successfully updated a notebook (id:2)
Initialize pickle_tutorial.ipynb
Error during notebook initialization. clean() got an unexpected keyword argument 'css_sanitizer'
Traceback (most recent call last):
  File "/Users/stroncoso/opt/miniconda3/envs/scheduler-env/lib/python3.10/site-packages/mercury/apps/notebooks/tasks.py", line 185, in task_init_notebook
    body = exporter.export(nb)
  File "/Users/stroncoso/opt/miniconda3/envs/scheduler-env/lib/python3.10/site-packages/mercury/apps/nb/exporter.py", line 136, in export
    body, _ = self.html_exporter.from_notebook_node(n)
  File "/Users/stroncoso/opt/miniconda3/envs/scheduler-env/lib/python3.10/site-packages/nbconvert/exporters/html.py", line 259, in from_notebook_node
    html, resources = super().from_notebook_node(nb, resources, **kw)
  File "/Users/stroncoso/opt/miniconda3/envs/scheduler-env/lib/python3.10/site-packages/nbconvert/exporters/templateexporter.py", line 427, in from_notebook_node
    output = self.template.render(nb=nb_copy, resources=resources)
  File "/Users/stroncoso/opt/miniconda3/envs/scheduler-env/lib/python3.10/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/Users/stroncoso/opt/miniconda3/envs/scheduler-env/lib/python3.10/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/Users/stroncoso/opt/miniconda3/envs/scheduler-env/share/jupyter/nbconvert/templates/lab/index.html.j2", line 4, in top-level template code
    {% from 'jupyter_widgets.html.j2' import jupyter_widgets %}
  File "/Users/stroncoso/opt/miniconda3/envs/scheduler-env/share/jupyter/nbconvert/templates/lab/base.html.j2", line 3, in top-level template code
    {% from 'cell_id_anchor.j2' import cell_id_anchor %}
  File "/Users/stroncoso/opt/miniconda3/envs/scheduler-env/share/jupyter/nbconvert/templates/base/display_priority.j2", line 1, in top-level template code
    {%- extends 'base/null.j2' -%}
  File "/Users/stroncoso/opt/miniconda3/envs/scheduler-env/share/jupyter/nbconvert/templates/base/null.j2", line 26, in top-level template code
    {%- block body -%}
  File "/Users/stroncoso/opt/miniconda3/envs/scheduler-env/share/jupyter/nbconvert/templates/base/null.j2", line 29, in block 'body'
    {%- block body_loop -%}
  File "/Users/stroncoso/opt/miniconda3/envs/scheduler-env/share/jupyter/nbconvert/templates/base/null.j2", line 31, in block 'body_loop'
    {%- block any_cell scoped -%}
  File "/Users/stroncoso/opt/miniconda3/envs/scheduler-env/share/jupyter/nbconvert/templates/base/null.j2", line 34, in block 'any_cell'
    {%- block codecell scoped -%}
  File "/Users/stroncoso/opt/miniconda3/envs/scheduler-env/share/jupyter/nbconvert/templates/lab/base.html.j2", line 13, in block 'codecell'
    {{ super() }}
  File "/Users/stroncoso/opt/miniconda3/envs/scheduler-env/share/jupyter/nbconvert/templates/base/null.j2", line 36, in block 'codecell'
    {%- block input_group -%}
  File "/Users/stroncoso/opt/miniconda3/envs/scheduler-env/share/jupyter/nbconvert/templates/lab/base.html.j2", line 22, in block 'input_group'
    {{ super() }}
  File "/Users/stroncoso/opt/miniconda3/envs/scheduler-env/share/jupyter/nbconvert/templates/base/null.j2", line 40, in block 'input_group'
    {%- block input -%}{%- endblock input -%}
  File "/Users/stroncoso/opt/miniconda3/envs/scheduler-env/share/jupyter/nbconvert/templates/lab/base.html.j2", line 30, in block 'input'
    {{ cell.source | highlight_code(metadata=cell.metadata) | clean_html }}
  File "/Users/stroncoso/opt/miniconda3/envs/scheduler-env/lib/python3.10/site-packages/nbconvert/filters/strings.py", line 90, in clean_html
    return bleach.clean(
TypeError: clean() got an unexpected keyword argument 'css_sanitizer'
```

**Comment by stroncod** [comment]
Fixed using `bleach 6.0.0`. Dependency requirements needs an update it seems

**Comment by pplonski** [comment]
Hi @stroncod,

Thanks for solving this. I've added `bealch>=6.0.0` in `requirements.txt`.
